src/api_client/index.js

```diff
@@ -13,6 +13,7 @@
  *
  */
 
-import liskAPI from './liskApi';
+import APIClient from './api_client';
 
-export default liskAPI;
+export default APIClient;
+export { getMainnetClient, getTestnetClient } from './api_client';
```

Comment:
Prefer the following syntax:
```
export {
    default as APIClient,
    getMainnetClient,
    getTestnetClient,
} from './api_client';
```

Reply:
I think this would have different result.
`APIClient` is not exported as default anymore

Reply:
Ah yeah, fair point.